config.go

```diff
@@ -122,6 +122,13 @@ type CORSConfig struct {
 	//
 	// Added in v4.2.0.
 	AllowAllOrigins bool
+
+	// AllowOrigins enables CORS and allows the list of origins in the
+	// HTTP request origins when set. Practically speaking, this
+	// results in the HTTP header "AllowOrigins".
+	//
+	// Added in v5.0.0.
+	AllowOrigins string
```

Comment:
I think this should be a slice of strings. While the header only can include 1 value, IIRC Gin will check the `Origin` header from the request and use the appropriate origin in the response header.

While we probably won't need more than 1, it's good to have the option available, especially since it's such an easy change but hard to change later as it changes the config schema.

Apparently Viper (that we're using for the configs) supports slices by separating elements via a comma.
(https://github.com/spf13/viper/blob/v1.7.1/viper.go#L920)

So you can change this to `[]string` and then add something like this to the godoc:
> Multiple origins can be supplied via environment variables by separating them by commas. YAML configuration is expected to use a regular array of strings.